Gruntfile.coffee

```diff
@@ -61,8 +37,9 @@ module.exports = (grunt) ->
         options:
           reporter: 'spec'
           require: [
+            # 'coffee-coverage/register-istanbul'
             'coffee-script/register'
-            'coffee-coverage/register-istanbul'
+            'babel-register'
```

Comment:
Tests breaks, because we babel-register had a [extension hook](https://nodejs.org/api/globals.html#globals_require_extensions) but it expect `.js` file.

A simpler approach could be compile our files before running test in a tmp dir, but it may breaks coverage and will be slower and dirtier.